packages/ts/generator-plugin-signals/test/fixtures/ValueSignalServiceMix.snap.ts

```diff
@@ -3,10 +3,24 @@ import { ValueSignal as ValueSignal_1 } from "@vaadin/hilla-react-signals";
 import type Person_1 from "./com/github/taefi/data/Person.js";
 import client_1 from "./connect-client.default.js";
 async function getPerson_1(init?: EndpointRequestInit_1): Promise<Person_1 | undefined> { return client_1.call("PersonService", "getPerson", {}, init); }
-function personSignal_1(): ValueSignal_1<Person_1 | undefined> {
-    return new ValueSignal_1(undefined, { client: client_1, endpoint: "PersonService", method: "personSignal" });
+function personSignal_1({ defaultValue: defaultValue_1 }: {
+    defaultValue: Person_1 | undefined;
+}): ValueSignal_1<Person_1 | undefined> {
+    return new ValueSignal_1(defaultValue_1, { client: client_1, endpoint: "PersonService", method: "personSignal" });
 }
-function personSignalNonNull_1(): ValueSignal_1<Person_1> {
-    return new ValueSignal_1(undefined, { client: client_1, endpoint: "PersonService", method: "personSignalNonNull" });
+function personSignalWithParams_1(dummyBoolean: boolean, dummyString: string | undefined, { defaultValue: defaultValue_2 }: {
+    defaultValue: Person_1 | undefined;
+}): ValueSignal_1<Person_1 | undefined> {
+    return new ValueSignal_1(defaultValue_2, { client: client_1, endpoint: "PersonService", method: "personSignalWithParams", params: { dummyBoolean, dummyString } });
 }
-export { getPerson_1 as getPerson, personSignal_1 as personSignal, personSignalNonNull_1 as personSignalNonNull };
+function personSignalNonNull_1({ defaultValue: defaultValue_3 }: {
+    defaultValue: Person_1;
+}): ValueSignal_1<Person_1> {
+    return new ValueSignal_1(defaultValue_3, { client: client_1, endpoint: "PersonService", method: "personSignalNonNull" });
+}
+function personSignalNonNullWithParams_1(dummyBoolean: boolean, dummyString: string | undefined, { defaultValue: defaultValue_4 }: {
+    defaultValue: Person_1;
+}): ValueSignal_1<Person_1> {
+    return new ValueSignal_1(defaultValue_4, { client: client_1, endpoint: "PersonService", method: "personSignalNonNullWithParams", params: { dummyBoolean, dummyString } });
+}
```

Comment:
Just thinking out loud (feel free to ignore, as it is not breaking anything):
Is the alias for `defaultValue` needed to be unique in the whole module? Isn't a function scoped uniqueness enough?